rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -333,6 +343,13 @@ declare export var MetadataJsonSchema: {|
 /**
  */
 
+declare export var StakeCredKind: {|
+  +Key: 0, // 0
+  +Script: 1, // 1
+|}
+
+/**
+ */
 declare export var CoinSelectionStrategyCIP2: {|
```

Comment:
I messed up the merge here. I'll remove the duplicate `CoinSelectionStrategyCIP2` in the next commit I do since it's a minor change.